GSoC-2025/Spikewrap:Extend Spikewrap Functionality(Ajitesh Kumar Singh).md

```diff
@@ -0,0 +1,221 @@
+# GSOC 2025 Application 
+
+
+## Extend Spikewrap Test Functionality
+
+## Personal details
+Please include the following information:
+- **Full name** : Ajitesh Kumar Singh
+- **Primary Email** :  Ajitesh.Kumar@iiitb.ac.in
+- **Secondary Email** : y.ajitesh710@gmai.com
+- **GitHub username** : Transyltooniaa
+- **Zulip username** : Ajitesh Kumar Singh
+- **Location & time-zone** India (UTC+5.30) 
+- **Personal website / project portfolio** (optional) : https://www.linkedin.com/in/ajitesh-kumar-singh-140529201/
+
+
+- **Code contribution**
+    - [Automate Sphinx Documentation Deployment with Versioned Switcher](https://github.com/neuroinformatics-unit/datashuttle/pull/486)  
+        Addresses [#372](https://github.com/neuroinformatics-unit/datashuttle/issues/372): Automates Sphinx docs deployment using GitHub Actions, deploying tagged releases to versioned directories, updating switcher.json, and maintaining the latest release.
+
+    - [Add option to delete a project](https://github.com/neuroinformatics-unit/datashuttle/issues/436)
+        Developed a function to delete both the DataShuttle project configuration and associated data from local and central paths. The implementation is currently on hold until it is requested by a user. But, the issue is left open and serves as a reference for future extensions. 
+
+    - **merged** : [Fixing json syntax in the mountainsort5 sorting yaml](https://github.com/neuroinformatics-unit/spikewrap/pull/228)  
+    While exploring the codebase and trying to understand it to better, I caught upon this minor syntax error and fixed it.
+
+    - [Completing Docstring for parameters of get_next_sub ](https://github.com/neuroinformatics-unit/datashuttle/pull/482)
+        Adding docstrings for the parameters of get_next_sub function. TDocstrings were added for the parameters of the get_next_sub function. This PR was prompted by a previous comment, leading to the opening of a related issue [#483](https://github.com/neuroinformatics-unit/datashuttle/issues/483)
+    
+    - [Implemented New Getters](https://github.com/neuroinformatics-unit/datashuttle/pull/480)
+    This PR introduces two new methods to streamline folder lookups within the datashuttle. Although the functionality of get_all_sub_and_ses_names which is now called get_all_sub_and_ses_paths was introduced in [#464]() which was soon to
+    be merged at that time, But since this was my first pull request , it help me understand the codebase.
+
+    - [Devlopment of Website for National Institute of Mental Health and Neuro Sciences](https://www.nimhans.ac.in/)
+    ![image](https://github.com/Transyltooniaa/gsoc/blob/MyProposal/GSoC-2025/Nimhans.png) 
+    A private Repository so cannot be shared
+
+- **Proposal discussion link**
+
+    Please link to the pull request where you discussed your project proposal with the community. 
+
+## Project proposal 
```

Comment:
In this section, can you be more specific about what you're going to test and how? It would help to show that you understand the package and the project. Currently most of the statements are variants on "add tests".